hbase-client/src/main/java/org/apache/hadoop/hbase/client/BufferedMutatorImpl.java

```diff
@@ -93,6 +99,9 @@
   private final boolean cleanupPoolOnClose;
   private volatile boolean closed = false;
   private final AsyncProcess ap;
+  private List<AsyncRequestFuture> asfList;
```

Comment:
Does this field need to be a class member? It seems that we only use it in the doFlush method.